src/formatDate.js

```diff
@@ -8,7 +8,46 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrayFromDate = date.split(fromFormat[fromFormat.length - 1]);
+  const arrForResulf = [];
+  const objectFromDate = {};
+
+  for (let i = 0; i < arrayFromDate.length; i++) {
+    objectFromDate[fromFormat[i]] = arrayFromDate[i];
+  }
+
+  const { MM: month, DD: day, ...lastElement } = objectFromDate;
+  const [year] = Object.values(lastElement);
```

Comment:
The extraction of the year from `objectFromDate` assumes that the year is always the last element. This might not be accurate if the `fromFormat` array does not have 'YYYY' or 'YY' as the last element. Consider using a more robust method to extract the year based on the format.